alfred-api-integrationtests-client/alfresco/src/main/java/eu/xenit/alfred/api/tests/JavaApiBaseTest.java

```diff
@@ -32,22 +31,17 @@ public abstract class JavaApiBaseTest extends BaseApplicationContextTest {
 
 
     protected NodeRef getNodeAtPath(String path) {
```

Comment:
This protected function replaces all the getNodeAtPath functions that were copied between the child classes.
(DRY, it's good this is centralized here or in a utils class.)

I cannot help but notice this function is slightly different from the removed ones.

It looks better but is it functionally the same?
One does select nodes the other does a query?